### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by AliAghel** [Planeshifter/node-wordnet-magic#20]
Hi @Planeshifter ,

Sadly there are couple of issues regarding installation of module including sqlite3 failure as well as downloading sqlite-31.db from provided link. (Using node v10.8.0)

```
node-pre-gyp ERR! Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v64-darwin-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for sqlite3@3.1.13 and node@10.8.0 (node-v64 ABI) (falling back to source compile with node-gyp) 
node-pre-gyp ERR! Tried to download(undefined): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v64-darwin-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for sqlite3@3.1.13 and node@10.8.0 (node-v64 ABI) (falling back to source compile with node-gyp) 
  ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c
  ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c
  TOUCH Release/obj.target/deps/action_before_build.stamp
  TOUCH Release/obj.target/deps/action_before_build.stamp
  CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o
  CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o
...
```

Apart from that, I managed to install your module locally and download the required database, but still got some errors such as:

`TypeError: wn.Word is not a constructor`
`TypeError: Cannot read property 'allAsync' of undefined`

Basically I'm interested to use `getSynsets`, `getHypernyms` and similar functions like your examples but got those error which I am not sure it would be related to installation issue or own module functionality.

Any chance to fix issues? Your module is the richest among other `wordnet` javascript/node modules and it would be a pity if you stop supporting it.

**Comment by queejie** [comment]
I have the same issue.  I am able to install sqlite3 version 4.0.9 without a problem, but it does not accept that.  Would it be possible to update this library?  Thank you!

**Comment by contactsamie** [comment]
@AliAghel  I ran into same issue, you will have to update sqlite , there is no other way, I did pull the lib to make the update , but in the end i striped out slite alltogether. the implementation is way behind this but it could get you hypernyms and synset https://www.npmjs.com/package/node-wordnet-magic-nosql . once this is officially fixed , I'll be back to using this , as you can go directly fron the sqlite db to get the synset data
